collect_app/src/main/res/values/strings.xml

```diff
@@ -12,20 +12,14 @@
 -->
 <resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en"
            tools:ignore="MissingTranslation">
-<string name="accept_location">Record Location</string>
```

Comment:
Given that we already have this and replace_location translated to a bunch of languages, perhaps we should use those instead of 'Start GeoPoint?' I think that would be a lot easier to understand. @yanokwa?